### PR TITLE
:zap: [RND-48793] Implement split button part 2

### PR DESCRIPTION
**Pull request by Gladkowski**

### CODE REVIEW

File: optimus/lib/src/button/button.dart [hunk 1]
```diff
@@ -4,6 +4,7 @@ import 'package:optimus/optimus.dart';
 import 'package:optimus/src/button/base_button.dart';
 import 'package:optimus/src/widget_size.dart';
 
+// TODO(VG): rename to OptimusButtonType
```
Comment (ookami-kb) [inline optimus/lib/src/button/button.dart:7]: I would say it's vice versa – according to documentation `OptimusSplitButtonType` should be renamed to `OptimusSplitButtonVariant`.

Comment (Gladkowski) [reply optimus/lib/src/button/button.dart]: Ok, will replace it in the task too

**Review by ookami-kb — CHANGES_REQUESTED** [review]
`size` is not working.

File: optimus/lib/src/button/split.dart [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+/// Split buttons have a main action and a dropdown action. The main action is
+/// on the left. An arrow on the right opens a dropdown menu with more actions
+/// related to the main action.
+class OptimusSplitButton<T> extends StatelessWidget {
+  const OptimusSplitButton({
+    Key key,
+    @required this.child,
+    this.onPressed,
+    @required this.items,
+    this.onDropdownItemPressed,
```
Comment (ookami-kb) [inline optimus/lib/src/button/split.dart:25]: Since we named it `onItemSelected` in dropdown button, let's go with the same naming here.

**Review by ookami-kb — APPROVED** [review]

**Pull request merged**